stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -327,6 +327,11 @@ public void request(int count) {
     public void setMessageCompression(boolean enable) {
       call.setMessageCompression(enable);
     }
+
+    @Override
+    public void cancel() {
```

Comment:
We want the `String, Throwable` arguments here. We already decided to remove the no-arg version on `ClientCall` and I see no reason to re-add it here.